content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -772,6 +772,12 @@ The Account Resource supports proprietary systems for:
 
 * Account.identifier.system is [Account Number](#account-number) for financial accounts.
 
+<h4>ChargeItem</h4>
+
+The ChargeItem Resource supports proprietary systems for:
+
+* ChargeItem.code, ChargeItem.reason, Modifier, Procedure, and Revenue Code extension codes are [Bill Codes Types](#bill-codes-types) for charge items.
```

Comment:
Would it more readable if we put 'Extension' after Modifier, Procedure, and Revenue Code to specify that those three are the extensions?

Reply:
+1

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/b18ef531da3e92fd88d55cd0da186b63db0671c1#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R779).
<img width="908" alt="Screen Shot 2021-04-07 at 3 03 08 PM" src="https://user-images.githubusercontent.com/32521644/113926941-718d7180-97b2-11eb-916d-44b486b06666.png">